packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -191,6 +191,10 @@ export function useTree(props: TreeProps, emit) {
     }
   }
 
+  function setExpendKeys(keys: TreeKey[]) {
```

Comment:
It seems there is a typo of `expand`. The wrong word `expend` is used here.

Reply:
Good catch! 👍

Reply:
fix in #9413